src/Corporation/Division.ts

```diff
@@ -553,6 +552,7 @@ export class Division {
             } else {
               sCost = mat.desiredSellPrice;
             }
+            mat.GUImarketPrice = sCost;
```

Comment:
Treat acronyms like regular words for camel-case purposes. So this should be `guiMarketPrice`, or maybe just `uiMarketPrice`.